source/portable/windows/core_pkcs11_pal.c

```diff
@@ -141,6 +143,13 @@ void prvLabelToFilenameHandle( uint8_t * pcLabel,
             *pcFileName = pkcs11palFILE_CODE_SIGN_PUBLIC_KEY;
             *pHandle = eAwsCodeSigningKey;
         }
+        else if( 0 == strncmp( pkcs11configLABEL_CMAC_KEY,
```

Comment:
Why not memcmp as in other conditions?

Reply:
I took these changes of AES CMAC support from the POSIX port of corePKCS11 PAL that uses `strncmp` (as can be seen [here](https://github.com/FreeRTOS/corePKCS11/blob/8a104f3e28a96f41813e67e5271ae756ea004e04/source/portable/posix/core_pkcs11_pal.c#L153-L158)). There is some duplication of functionality between the Windows and POSIX ports, something that can be hygiened into a common file to reduce maintenance cost.